tests/components/technove/test_sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for the TechnoVE sensor platform."""
+from unittest.mock import MagicMock
+
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default", "mock_technove")
+async def test_sensors(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test the creation and values of the TechnoVE sensors."""
+    mock_config_entry.add_to_hass(hass)
+
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
+    entity_registry = er.async_get(hass)
+    entity_entries = er.async_entries_for_config_entry(
+        entity_registry, mock_config_entry.entry_id
+    )
+
+    assert entity_entries
+    for entity_entry in entity_entries:
+        assert hass.states.get(entity_entry.entity_id) == snapshot(
+            name=entity_entry.entity_id
+        )
```

Comment:
It's also nice to test the entity entry itself as well. Maybe checkout the Glances sensor tests

Reply:
Done.